.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "test/web-platform-tests/tests"]
 	path = test/web-platform-tests/tests
 	url = https://github.com/web-platform-tests/wpt.git
+[submodule "benchmark/selectors/sizzle"]
```

Comment:
Since submodules are already a pain, and I don't think we'll need to update this one very often, I think it'd be best to just directly check in the two relevant files. We can add a or modify a README.md in an appropriate directory to explain where they came from.

Reply:
Done.
I'll leave the README up to you.